theano/gof/opt.py

```diff
@@ -335,6 +333,10 @@ def print_profile(stream, prof, level=0):
                 print(blanc, '  %.6fs - %s' % (t, opt), file=stream)
 
             if sub_profs[i]:
+                if isinstance(opts[i], MergeOptimizer):
+                    if len(sub_profs[i]) == 8:
+                        sub_profs[i] = sub_profs[i][:-1]
+                    assert len(sub_profs[i]) == 7
```

Comment:
I think this diff isn't needed anymore. I did another fix that don't require this hack.